### PR TITLE
2d labelled points convenience function

### PR DESCRIPTION
**Pull request by pkerpedjiev-zymergen**
## Description

Added a convenience function for loading 2d labelled points from a pandas dataframe.

Why is it necessary?

To avoid the much longer method of declaring a tileset info and tile loader.

Fixes #___

## Checklist

- [x] Unit tests added or updated
- [ ] Documentation added or updated
- [x] Updated CHANGELOG.md

### CODE REVIEW

File: higlass/tilesets.py [hunk 1]
```diff
@@ -5,8 +5,10 @@
 
 import clodius.tiles.utils as hgut
```
Comment (nvictus) [inline higlass/tilesets.py]: These imports are getting out of hand and are unreadable imho. Maybe they should be migrated into the functions that use them.

File: test/tilesets_test.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+def test_dfpoints():
+    df = pd.DataFrame({
+        'a': [1.0, 2.0, 3.0],
+        'b': [5.0, 6.0, 7.0],
+        'x': ['hi', 'there', 'you']
+    })
+
+    ts = hgti.dfpoints(
+        df, 'a', 'b', 'x'
```
Comment (nvictus) [inline test/tilesets_test.py]: I'm confused. Does the `x` passed to `dfpoints` refer to a uuid or a column of `df`?

File: higlass/tilesets.py [hunk 3]
```diff
@@ -100,6 +102,34 @@ def mrmatrix(filepath, uuid=None):
...
+    tsinfo = hgpo.tileset_info(df, x_col, y_col)
+
+    return Tileset(
+        tileset_info=lambda: tsinfo,
+        tiles=lambda tile_ids: hgpo.format_data(
+                    hgut.bundled_tiles_wrapper_2d(tile_ids,
+                        lambda z,x,y,width=1,height=1: hgpo.tiles(df, x_col, y_col,
+                            tsinfo, z, x, y, width, height))))
+
 import clodius.tiles.nplabels as ctnl
```
Comment (nvictus) [inline higlass/tilesets.py]: These should be migrated to the top if kept.

**Comment by pkerpedjiev** [comment]
Thanks for the comments! I was actually working on this at home and made some changes (including your suggestions) from my non-Zymergen account in this PR: https://github.com/higlass/higlass-python/pull/18

**Pull request closed without merging**